### ISSUE TITLE
Request - config option for how much extra health the mud armor gives

### ISSUE THREAD

**Issue by katubug** [Liahim85/SaltyMod#6]
Hi! I am tweaking things for my modpack, and I was surprised to see that the mud armor increases HP by almost double! I was hoping to offer it as an early-midgame armor option, but this makes it quite powerful. I don't really want to have to make the recipe more expensive, but I don't see a way to change the HP increase in the config file.

I suppose I could reduce the durability more, but my ideal solution would be to just decrease the HP added by each piece.

By the way, I think the fact that the armor degrades in the water and the rain is fantastic, I love that! Thank you for your time, have a lovely weekend!

**Comment by Liahim85** [comment]
As you know, I can't work on salty until I finish the MW update. After that I'll try to do something :)

**Comment by katubug** [comment]
Yep, no worries at all! Thank you very much for your consideration <3
